test/common/index.js

```diff
@@ -737,14 +737,15 @@ function invalidArgTypeHelper(input) {
     return ` Received function ${input.name}`;
   }
   if (typeof input === 'object') {
-    if (input.constructor && input.constructor.name) {
+    if (input?.constructor?.name) {
```

Comment:
```suggestion
    if (input.constructor?.name) {
```

We know for certain that `input` is an object of some kind (null is already checked above).